src/components/FeatureViews/AdminPortal/AdminUtils.js

```diff
@@ -0,0 +1,173 @@
+import { isSunday } from "date-fns";
+
+export const roleColor = (string)=>{
+    switch(string){
+        case "administrator":
+            return "red";
+        case "parent":
+            return "#812CD7";
+        case "student":
+            return "#378BFF";
+        case "instructor":
+            return "#1FA18A"
+        case "receptionist":
+            return "#D04E3D"
+        default:
+            return "no match"
+    }
+}
+
+export const statusColor = (x)=> {
+    switch(x){
+        case "0":
+            return "red";
+        case "1":
+            return "red";
+        case "2":
+            return "yellow";
+        case "3":
+            return "yellow";
+    }
+}
+
+export const initials = (first, last) => {
+    if (first) {
+        let x = 0;
+        let firstI = first.charAt(0);
+        if (firstI.match(/[A-Z]/)){
+        let firstI = first.charAt(0)
+        let lastI = last.charAt(0)
+        return firstI + lastI
+        }else{
+            let firstI = first.charAt(1)
+            let lastI = last.charAt(0)
+            return firstI + lastI
+        }
+    }
+}
+
+export const emailCheck= (x) => {
+    if (x){
+        return x
+    }
+    else {
+        return "Email n/a"
+    }
+
+}
```

Comment:
This can be concisely written as:

    export const emailCheck = (email) => email || "Email N/A";

This will return the first "truthy" value it encounters: if the email is defined, that is returned, otherwise "Email N/A" is returned.